gateway-export-plugin/src/test/java/com/ca/apim/gateway/cagatewayexport/CAGatewayExportTest.java

```diff
@@ -107,6 +108,50 @@ void testExplode(TemporaryFolder temporaryFolder) throws IOException, URISyntaxE
         assertEquals(1, passwordProperties.size());
     }
 
+    @Test
+    @ExtendWith(TemporaryFolderExtension.class)
+    void testExplodeWithMissingEntities(TemporaryFolder temporaryFolder) throws IOException, URISyntaxException {
+        String projectFolder = "example-project";
+        File testProjectDir = new File(temporaryFolder.getRoot(), projectFolder);
+        File buildGradleFile = new File(testProjectDir, "build.gradle");
+        File bundleFile = new File(testProjectDir, "bundle.bundle");
```

Comment:
This should be `missing-entities-test.bundle` file

Reply:
done